packages/language/test/parser/pli-preprocessor-include.test.ts

```diff
@@ -262,4 +262,40 @@ describe("PL/1 Includes with Plugin Config", () => {
       ";:;",
     ]);
   });
+
+  test("Include with XInclude", () => {
```

Comment:
Suggestion: These tests should a bit better describe what they're trying to do:
```suggestion
  test("Should include only once with 2 XINCLUDE statements", () => {
```